pagopa/pn-frontend
# Chore: transform taxId in uppercase before notification creation

## Short description
transform taxId in uppercase before notification creation.

## List of changes proposed in this pull request
- Added taxId transformation in newNotificationRecipientsMapper() function

## How to test
Create a new notification and insert tax id in lowercase then check the payload of API notification creation. You'll see the taxId in uppercase
